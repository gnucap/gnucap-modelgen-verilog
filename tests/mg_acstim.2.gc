
load ./modelgen_0.so

verilog

`modelgen
module acstim2(p, n);
	electrical p, n;
	inout p, n;
	analog begin
		I(p,n) <+ ac_stim("ac", 2.);
		I(p,n) <+ ac_stim("ac", 0.);
	end
endmodule


!make acstim2.so > /dev/null
attach ./acstim2.so

parameter v=0

module mm(1, gnd)
	acstim2 d(1,gnd);
	resistor #(1.) r(1,gnd);
endmodule

mm m1(a, 0);

print dc v(nodes) x(m1.d.*)
dc

print ac v(nodes)
ac 1 100 * 10
status notime
end
