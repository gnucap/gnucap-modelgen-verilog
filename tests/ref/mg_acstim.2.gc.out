#           v(a)       x(m1.d._b_p_n) x(m1.d._b_acs_0_p_acs_0_n)
 0.         0.         ??         ??        
#Freq       v(a)      
 1.         2.        
 10.        2.        
 100.       2.        
Gnucap   System status
iterations: op=0, dc=3, tran=0, fourier=0, total=6
transient timesteps: accepted=0, rejected=0, total=0
nodes: user=1, subckt=0, model=0, total=1
dctran density=100.0%, ac density=100.0%
